src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -359,83 +359,84 @@ public override Stream EndGetRequestStream(IAsyncResult asyncResult)
                 throw new InvalidOperationException(SR.net_reqsubmitted);
             }
 
-            var handler = new HttpClientHandler();
-            var client = new HttpClient(handler);
-            var request = new HttpRequestMessage(new HttpMethod(_originVerb), _requestUri);
-
-            if (_requestStream != null)
+            using (var handler = new HttpClientHandler())
```

Comment:
It is not necessary to have a `using` for the handler.  Disposing the `HttpClient` will dispose the inner handler automatically.  So, it's only necessary to have the `using` for the `HttpClient` itself.